web/themes/custom/cloudy/scss/components/_card.scss

```diff
@@ -1,19 +1,18 @@
 .node--type-city-service {
-  .card:not(:first-child) {
-    max-width: calc(50% - 30px);
-  }
   .card {
-    color: red;
-    flex-basis: calc(50% - 30px);
-      @include media-breakpoint-up(sm) {
-        margin-bottom: $card-deck-margin;
-      }
-      @include media-breakpoint-only(md) {
-        flex-basis: 100%;
-        flex-direction: row;
-      }
-    @include media-breakpoint-up(lg) {
-      flex-wrap: wrap;
+    &:first-child {
+      max-width: 100%;
+    }
+
+    @include media-breakpoint-up(sm) {
+      flex-basis: calc(50% - 30px);
+      max-width: calc(50% - 30px);
```

Comment:
The 30px is a magic number, and can be generated based off of the card-deck-margin variable you use below.